phial/wrappers.py

```diff
@@ -109,11 +109,20 @@ def __init__(self,
                  channel: str,
                  text: Union[str, None] = None,
                  original_ts: Union[str, None] = None,
+                 attachments: Union[object, None] = None,
                  reaction: Union[str, None] = None) -> None:
         self.channel = channel
         self.text = text
         self.original_ts = original_ts
         self.reaction = reaction
+        self.attachments = attachments
+
+    def serialiseAttachments(self) -> list:
```

Comment:
This feels a bit wrong to me, mutating an existing list and changing all the elements, it also means that we can't have the advantages of static types.

I feel like `self.attachments` should be of type `Optional[List[MessageAttachment]]` and that the conversion to JSON should happen elsewhere, or be immutable (return the json).

Reply:
Also this should be `snake_case` to conform with PEP8